src/test/java/com/thealgorithms/others/ArrayRightRotationTest.java

```diff
@@ -0,0 +1,17 @@
+package com.thealgorithms.others;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ArrayRightRotationTest {
+
+    @Test
```

Comment:
add different test cases. You can reference this code.
[https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/others/ArrayLeftRotationTest.java](url)